docs/translation.md

```diff
@@ -43,15 +48,21 @@ sequenceDiagram
   Developer ->> GitHubActions: Merge pull request into develop
   GitHubActions ->> GitHubActions: Scan codebase and write strings to .po file
   GitHubActions ->> GitHubActions: Fill .po file with existing translations
-  GitHubActions ->> GitHub: Commit .po file with updated strings
-  GitHubActions ->> Poeditor: Call webhook
-  Poeditor ->> GitHub: Fetch updated .po file
-  Poeditor ->> Poeditor: Synchronize translations with latest strings and translations
-  Translator ->> Poeditor: Translate strings
-  Translator ->> Poeditor: Export strings to GitHub
-  Poeditor ->> GitHub: Commit .po file with updated translations to develop
+  Note over GitHubActions,GitHubActions: if config translations<br/>are available<br/>they are used<br/>otherwise empty strings
```

Comment:
```suggestion
  Note over GitHubActions: If config translations are available<br/>they are used. Otherwise empty strings.
```

A few things here:

1. Capitalization :smile:
2. Lines in the note are very short. I think they could be made a tad longer.
3. I wonder why the actor is repeated

Reply:
1. Yes
2. It is because how Mermaid is rendering the note. Looks better like this
3. Because we have a GitHubActions -> GitHubActions flow